website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -74,6 +76,7 @@ This example creates a time spine at an hourly grain and a daily grain: `time_sp
 
 <Lightbox src="/img/time_spines.png" width="50%" title="Time spine directory structure" />
 
+<!--
```

Comment:
Is this commenting out the version block below?

Reply:
yes it is! but we dont need this yaml for 1.8 and lower right?